CMakeLists.txt

```diff
@@ -263,7 +263,7 @@ ENDIF()
 MY_CHECK_AND_SET_COMPILER_FLAG("-Wno-deprecated-copy" DEBUG RELEASE RELWITHDEBINFO MINSIZEREL)
 MY_CHECK_AND_SET_COMPILER_FLAG("-Wno-deprecated-declarations" DEBUG RELEASE RELWITHDEBINFO MINSIZEREL)
 
-MY_CHECK_AND_SET_COMPILER_FLAG("-Werror -Wall -Wextra")
```

Comment:
Does your patch involve any inevitable warnings?

Reply:
This is still WIP, vigilantee )